### PR TITLE
#4155 Card leftovers

### LINKED ISSUE #4155: Redesign: Card leftovers
### Is your feature request related to a problem?

image
the card breaks when there is no collection/nft name. Also, In this case, it would be great to make max name length and then only add 3 dots behind if it is over.

### Describe the solution you would like

.

### Describe alternatives you have considered

.

### Screeenshots

_No response_

### Are there opened related issues?

_No response_

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

### What's new?

- [x] PR closes #4155
- [ ] Requires deployment <rubick/magick_issue>
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

**Comment by prachi00** [comment]
I think that its not possible to have an empty collection name because its required in the create collection page?

**Review by roiLeo — COMMENTED** [review]
Hey! what did you change?

Screenshot 2022-10-21 at 08-39-31 KodaDot - Kusama NFT Market Explorer

> the card breaks when there is no collection/nft name. Also, In this case, it would be great to make max name length and then only add 3 dots behind if it is over.

you could try to use `text-overflow: ellipsis;`

**Comment by prachi00** [comment]
> Hey! what did you change?
> 
> Screenshot 2022-10-21 at 08-39-31 KodaDot - Kusama NFT Market Explorer
> 
> > the card breaks when there is no collection/nft name. Also, In this case, it would be great to make max name length and then only add 3 dots behind if it is over.
> 
> you could try to use `text-overflow: ellipsis;`

yeah so thois class was already given, I just had to remove some things so it works fine now

**Comment by roiLeo** [comment]
Can you explain what was changed in this pull request?

**Comment by prachi00** [comment]
Before: 
Screen Shot 2022-10-24 at 6 36 09 PM


After: 
Screen Shot 2022-10-24 at 6 43 06 PM


so long texts show ellipsis

File: components/carousel/module/CarouselInfo.vue [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
+  <div class="carousel-info is-flex is-flex-direction-column">
     <nuxt-link
       :to="urlOf({ id: item.id, url, chain: item.chain })"
       :class="[
-        'has-text-weight-bold',
+        'has-text-weight-bold carousel-info-name',
         { 'carousel-info-collection': isCollection },
       ]">
-      <span class="carousel-info-name">{{ item.name }}</span>
+      <span class="carousel-info-name" :title="item.name">{{ item.name }}</span>
```
[the remark this answers is not in the document]
Comment (prachi00) [reply components/carousel/module/CarouselInfo.vue]: done

**Review by roiLeo — APPROVED** [review]
✅ code lgtm

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $40 @ 35.44 USD/KSM ~ 1.129 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0x64326e89280f47b25feaa10c5c31a7a9fc78aa812df970cebab57fbc990d9920](https://kusama.subscan.io/extrinsic/0x64326e89280f47b25feaa10c5c31a7a9fc78aa812df970cebab57fbc990d9920)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**